test/integration_test_helper.rb

```diff
@@ -299,7 +344,7 @@ def database_cleaner_strategy
   end
 
   def login_admin
-    visit('/users/login') if Capybara.current_driver == :selenium_chrome
+    visit('/users/login') if Capybara.current_driver.to_s.include? "chrome"
```

Comment:
I wonder if this shouldn't be always done for any selenium browser instead of chrome.
```suggestion
    visit('/users/login') if Capybara.current_driver.to_s.include? "selenium"
```

Reply:
Good point, adapted.